src/Queue/Worker.php

```diff
@@ -27,6 +27,9 @@ public function runSqsJob(Job $job, string $connectionName, WorkerOptions $optio
                 $job,
                 $this->maxAttemptsExceededException($job),
             );
+
+            // exit so that PHP will shutdown and close DB connections etc.
+            exit(1);
```

Comment:
This is after the job has been "failed" and thus the job has already been released back to SQS and it _should_ also have been added to the `failed_job` table.